File: individual-projects/2023/amoli-patel-200103102100/Fundamental Practice/loop/lift.js
// Assume 14 floor building, and Today Lift is not working and you are in building to repair it. Go through stairs and check where is the lift, repair it and come back in lift.

function main() {
    let person = 0,
        lift = 4,
        total_floors = 14
    for (check = 0; check <= total_floors; check++) {
        if (lift === person) {
            console.log(check, 'floor.....Got lift now repair')
            for (lift = check; lift >= 0; lift--) {
                console.log(lift, 'floor.....Coming down')
            }
            break
        } else {
            console.log(check, 'floor.....Going up')
            person++
        }
    }
}
main()

File: individual-projects/2023/akshit-vadodariya-200103101045/Loops/lift.rs
fn main() {
    let mut j = 0;
    let lift = 5;
     for _i in 0..14 {
        if lift == j {
            println!("Repair it and come back in lift.");
            while true {
                if j < 14 && j > 0  {
                    println!("Back to ground floor using lift {}", j);
                    j-=1;
                }
            }
            break;
        }
        else {
            println!("Go one floor up {}", j);
            j+=1;
        }
     }
}






fn main() {
    let mut j = 0;
    let lift = 5;
     for _i in 0..14 {
        if lift == j {
            println!("Repair it and come back in lift.");
            for _i in 0..14 {
                if _i < lift{
                    println!("Back to ground floor using lift {}", j);
                    j-=1;
                }
            }
            break;
        }
        else {
            println!("Go one floor up {}", j);
            j+=1;
        }
     }
}